File: class work/28-8-2024/task1.cpp
#include<iostream>
using namespace std;
class  Bank{
	public:
		string name;
		int acno,balance=5000;
		int depo,wth;
		acopen()
		{
			cout<<"Name: ";
			cin>>name;
			cout<<"Ac no: ";
			cin>>acno;
			
			cout<<"Deposit is : "<<balance<<endl;
			
			this-> balance = balance;
		}
		
		deposit()
		{
			int depo;
			cout<<"Enter amount for deposit: "<<endl;
			cin>>depo;
			
			this->depo=depo;
			
			this->balance=this->balance+this->depo;
			
			cout<<"Now your balance is: "<<this->balance<<endl;
		}
		
		withdraw()
		{
			int wth;
			cout<<"Enter Amount for withdraw"<<endl;
			cin>>wth;
			
			this->wth=wth;
			
			this->balance= this->balance-this->wth;
			
			cout<<"Now your balance is: "<<this->balance<<endl;
			
			
		}
};


main()
{
	Bank obj;
	obj.acopen();
	obj.deposit();
	obj.withdraw();
}

File: class work (python)/class work (python)/pacakes/task1.py
d = {}
import random
ac_no=random.randint(100000000000001 , 900000000000009)
balance = 5000

class bank: 
    def register(self):
        name=input("Enter your name : ")
        email=input("Enter email : ")
        password = int(input("Enter Password :"))
        cpassword = int(input("Enter Confirm Password :"))

        self.balance=balance
        if password==cpassword:
            print("Register Sucessfully!!")

            print("Your generated accout number is : ",ac_no)
            print("Your balance is : ",balance)

            d['name']=name
            d['email']=email
            d['password']=password
            d['ac_no']=ac_no

        else:
            print("password and confirm password dioes not match!!")
        

    def log(slef):
        ac_no=input("Enter accpont number : ")
        password=input("Enter password : ")
        print("Your balance is : ",balance)
        if d['ac_no']==ac_no:

            if d['password']==password:
                print("Login sucessfully!!")

            else:
                print("Invalid password!!")
        else:
            print("Invalid account number!!")

    
    def we(self):
        wamount=int(input("Enter wirhdraw amount : "))

        if wamount>balance:
            print("Invalid balance!!")

        else:
            self.balance = self.balance-wamount
    
    def de(self):
        damount=int(input("Enter deposit amount : "))

        self.balance=self.balance-damount

    # def check(slef):
    #     print("Your available balance is  : ",self.balance)

    def check(self):
        print('Your available balance is : ',self.balance)

obj=bank()
